code/modules/awaymissions/mission_code/Cabin.dm

```diff
@@ -111,8 +111,8 @@
 	color = rgb(0,0,255)
 
 /obj/structure/ladder/unbreakable/rune/Initialize()
-	. = ..()
 	AddElement(/datum/element/update_icon_blocker)
+	return ..()
```

Comment:
``update_icon_blocker`` should be called before parent since sometimes Initialize() will call update icon. Even if it isn't always the case, I wanted to make it consistent for them all so people looking to use it know how it should be used.